templates/CONTRIBUTING.tpl.md

```diff
@@ -1,144 +0,0 @@
-**VARS TO REPLACE IN TEMPLATE:**
```

Comment:
Well, cozy-template is a template for new client-side apps. And there is some work to do on cozy-template, including updating the README and CONTRIBUTING.

But these files are here because they are also useful for other contexts: the libs for example.